### ISSUE TITLE
Incorrect maximum on accessibility graph

### ISSUE THREAD

**Issue by mattwigway** [conveyal/analyst-server#183]
The maximum of the accessibility graph is incorrect. It should be the maximum worst case, not the maximum best case.

**Comment by mattwigway** [comment]
fixed upstream by opentripplanner/OpenTripPlanner#2110.
